news/editions/edition4.md

```diff
@@ -0,0 +1,81 @@
+
+## [Continuous deployment to Kubernetes using GitHub-triggered Spinnaker pipelines](https://blog.opsmx.com/continuous-deployment-to-kubernetes-using-github-triggered-spinnaker-pipelines/)
+Learn to configure a webhook to trigger pipelines in Spinnaker 1.19.1 based on commits to a GitHub repository. Set up Spinnaker to listen to changes in a GitHub artifact repository, inject changed GitHub files as artifacts into your pipeline, and provide execution verification.
+
+## [Spinnaker – Configuring dynamic Kubernetes accounts using Vault](https://blog.opsmx.com/spinnaker-configuring-dynamic-kubernetes-accounts-using-vault/)
+We can configure Spinnaker's Clouddriver with external configuration stores, such as HashiCorp Vault, a secrets management tool, to keep Kubernetes account information secure outside of Spinnaker. Consult this guide and its [sequel](https://blog.opsmx.com/spinnaker-externalising-kubeconfig-files-of-kubernetes-accounts/) to learn about externalizing dynamic Kubernetes accounts and kubeconfig files in Vault.
+
+## [Spinnaker Operator Demo](https://www.youtube.com/watch?v=VojwvcxZF3k)
+Take a look at the Spinnaker Operator, a Kubernetes operator for Spinnaker that makes it easier to install, deploy, and upgrade any version of Spinnaker using standard Kubernetes application workflows and tools, like kubectl, helm, or kustomize.
+
+## [Spinnaker idea: reusable run job stages via script runner containers](https://medium.com/@tomas_lin/spinnaker-idea-reusable-run-job-stages-via-script-runner-containers-ff5fd95ec056)
+Combine the deployment ease of the run script stage with the security advantages of a containerized stage by building general runner containers that run mix-and-match script workflows from S3. Enjoy the flexibility and security of a container runtime, and reduce time spend building containers.
+
+## [A Guide On The Installation Of Spinnaker in Kubernetes Clusters](https://www.magalix.com/blog/a-guide-on-the-installation-of-spinnaker-in-your-production-kubernetes-cluster)
+With this guide, use Halyard to install Spinnaker in a Kubernetes cluster, such as one you create in Google Kubernetes Engine. Configure an IAM role, storage bucket, and Halyard, and deploy a production-grade instance of Spinnaker.
+
+## [Spinnaker projects participating in Google Summer of Code](https://cd.foundation/blog/2020/05/18/9-cd-foundation-projects-are-participating-in-this-years-google-summer-of-code/)
+This year, we have joined GSoC to bring more student developers into the Spinnaker community. We'll nurture two Spinnaker projects: "Drone CI type for Spinnaker pipeline stage," from Victor Odusanya mentored by Armory engineer [Cameron Motevasselani](https://www.youtube.com/watch?v=HtkXeC8a38Y); and “Continuous Delivery, Continuous Deployments with Spinnaker,” from Moki Daniel mentored by Armory engineer [Fernando Freire](https://www.armory.io/blog/identifying-risk-when-executing-your-kubernetes-migration/).
```

Comment:
```suggestion
This year, we joined GSoC to bring more student developers into the Spinnaker community. We're nurturing two Spinnaker projects: "Drone CI type for Spinnaker pipeline stage" from Victor Odusanya (mentored by Armory engineer [Cameron Motevasselani](https://www.youtube.com/watch?v=HtkXeC8a38Y)) and “Continuous Delivery, Continuous Deployments with Spinnaker” from Moki Daniel (mentored by Armory engineer [Fernando Freire](https://www.armory.io/blog/identifying-risk-when-executing-your-kubernetes-migration/)).

Reply:
Also, the links for Cameron and Fernando seem kid of random

Reply:
Agree, rm'd